packages/edit-site/src/components/layout/router.js

```diff
@@ -148,15 +150,24 @@ export default function useLayoutAreas() {
 	}
 
 	// Styles
-	if ( path === '/wp_global_styles' ) {
+	if ( path && path.startsWith( '/wp_global_styles' ) ) {
 		return {
-			key: 'styles',
+			key: 'default',
 			areas: {
 				sidebar: (
 					<SidebarNavigationScreenGlobalStyles backPath={ {} } />
 				),
-				preview: <Editor />,
-				mobile: hasEditCanvasMode && <Editor />,
+				content: <GlobalStylesUIWrapper />,
+				preview: <Editor isPreviewOnly />,
```

Comment:
For some reason, when I navigate to the "styles" menu, the frame doesn't animate anymore (you don't see it moving to the right), I'm pretty sure it used to animate earlier in the PR

Reply:
> I think removing the drill-down removes the animation of the panel opening, I think we can not easily avoid having the drill down and also have the animation when the panel opens.

Why, how is the drill down related to the animation?

Reply:
I can easily re-add the animation by changing key from default to styles:
```
	if (path && path.startsWith('/wp_global_styles')) {
		return {
			key: 'styles',
```

The problem is that doing this doesn't just animate the style panel, but also animates the main "Design" area, which feels strange because there is no change in that part, only the styles item gets the blue background.

It seems that we can either animate the "Design" section and the style area, or we don't animate any of the areas. I don't see an option to not animate the style area but animate the styles panel.

Reply:
Maybe it's better to avoid animating based on the key, but let's not get blocked with that here, we can look at the animation separately.